src/components/AuthenticatedApp/AuthenticatedApp.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import HomePage from 'pages/HomePage';
+import NoMatchPage from 'pages/NoMatchPage';
+import SettingsPage from 'pages/SettingsPage';
+
+const AuthenticatedApp = () => (
+  <Switch>
+    <Route path="/" exact>
+      <HomePage />
+    </Route>
+    <Route path="/settings" exact>
+      <SettingsPage />
+    </Route>
+    <Route>
+      <NoMatchPage />
```

Comment:
nomatch should be on both "apps" right?

Reply:
I did it on purpose, given we do not have public pages (aside auth screens). The routes are divided. Thus, it could be confusing for an unauthenticated user to navigate to '/ account' and get a 404 error. IMO, redirecting to the login screen gives a better UX.

Reply:
oh, I didn't notice that now we will redirect to login in this case. I like it 👌